operator/internal/manifests/internal/config/loki-runtime-config.yaml

```diff
@@ -54,6 +54,23 @@ overrides:
     {{- if $spec.QueryLimits.CardinalityLimit }}
     cardinality_limit: {{ $spec.QueryLimits.CardinalityLimit }}
     {{- end }}
+    {{- with $l.Blocked }}
+    blocked_queries:
+    {{- range $blockedQuery := . }}
+    - {{ with $blockedQuery.Pattern -}}
```

Comment:
I think we need to add another validation to the LokiStack: all attributes of `BlockedQuerySpec` are optional, because depending on the use-case you only need a few of them. But our config-generation depends on _at least one_ of the attributes being set, otherwise we produce invalid YAML.

Would it make sense to add a validation to the LokiStack that emits an error if there is a `BlockedQuerySpec` which contains no values? Example:

```yaml
spec:
  limits:
    tenants:
      application:
        queries:
          blocked:
          - {}
```

Reply:
Ok, small correction: The generated YAML does not seem to be invalid, like I thought. So it's not technically needed to validate this. Is it still a good idea, though?